<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Lukin Lab - B26 GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="0">
      <property name="margin">
       <number>1</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_1" stretch="0,1,26,0,0,0">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="data_filepath"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_open">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_run">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QWidget" name="toolbar_space" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QWidget" name="plot" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="peak_locs"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="input_next"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_goto">
            <property name="text">
             <string>Go To</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_prev">
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_skip">
            <property name="text">
             <string>Skip</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_fit">
            <property name="text">
             <string>Fit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_clear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_next">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="btn_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="btn_load_gui">
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="btn_save_gui">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="btn_about">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="btn_test_2">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
